### DISCUSSION TITLE
Two different jest globals fixes documented in two places under https://mswjs.io/docs/

### DISCUSSION THREAD

**Discussion by jg210** [mswjs/msw#2034] [General]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 18 or higher

### Node.js version

N/A

### Reproduction repository

N/A

### Reproduction steps

The online docs offer two different ways of solving problem of jest removing globals for fetch, Headers, Request, Response, FormData,    Blob etc.:

https://mswjs.io/docs/faq#requestresponsetextencoder-is-not-defined-jest
https://mswjs.io/docs/runbook#fixing-environment

### Current behavior

Two ways to solve one problem, documented in two places. I didn't find the runbook link at first.

### Expected behavior

* Pick one solution and use it in both places?
* Mention both solutions in both places? 
* Merge FAQ and runbook pages? 
* Link to some checked-in jest config used by some msw automated tests, so can be more certain it works?

**Comment by kettanaito** [comment]
Hi, @jg210. That's a great suggestion. I didn't realize we were suggesting different things across different pages. We are already reusing the same Jest setup recommendations across [Migration guidelines](https://mswjs.io/docs/migrations/1.x-to-2.x#requestresponsetextencoder-is-not-defined-jest) and the [FAQ](https://mswjs.io/docs/faq#requestresponsetextencoder-is-not-defined-jest) pages. I will add it to the Runbook page as well.

**Reply by kettanaito** [reply]
I've updated the [Runbook](https://mswjs.io/docs/runbook#fixing-environment) and it now features the same Jest globals recommendations as everywhere else in the docs.
